src/server/worldserver/worldserver.conf.dist

```diff
@@ -1758,6 +1758,14 @@ ListenRange.TextEmote = 40
 
 ListenRange.Yell = 300
 
+#
+#    Creature.MovingStopTimeForPlayer
+#        Description: Time (in milliseconds) during which creature will not move after
+#                     interaction with player.
+#        Default: 180000
+
+Creature.MovingStopTimeForPlayer = 180000
```

Comment:
It's never implemented in the core.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/blob/9e0e4acc61a33159d6e737c54bf2f840223e2c83/src/server/game/World/World.cpp#L1177